resources/app/src/components/people/DemographicsForm.vue

```diff
@@ -716,7 +716,7 @@ export default {
                 reside_country_other: null,
                 reside_country_opt_out: false,
                 reside_state: null,
-                reside_state_opt_out: false,
+                state_opt_out: false,
```

Comment:
Why make this variable name inconsistent? Others have the pattern of {item}, {item}_other, {item}_opt_out... so it would make the most sense to be reside_state_opt_out...

Reply:
Good point.  At one time I used the existing state field from the profile.  The better approach is to keep the demographic information separate.